### PR TITLE
Added Guest demographic pie charts

### PR DESCRIPTION
**Pull request by townsend3721**

### CODE REVIEW

File: features/guests_added.feature [hunk 1]
```diff
@@ -34,4 +65,26 @@ Feature: See if you are attending an event
...
+      | John           | Doe              | jd@gmail.com | Undergrad            | Male               | Chemistry           | 1       |
+      | Sara           | Sen              | ss@gmail.com | Graduate             | Female             | Chemical Engineering| 1       |
+      | Steph          | Curry            | sc@gmail.com | Postdoc              | Male               | Stem non-chemistry  | 1       |
+      | Tara           | Gon              | tg@gmail.com | Faculty Staff        | Prefer not to say  | Other               | 1       |
+      | Alex           | Morgan           | am@gmail.com | Other                | Female             | Other               | 1       |
+      | Sasha          | Downer           | sd@gmail.com | Postdoc              | Other              | Stem non-chemistry  | 1       |
+      
+      
+  When I follow "event_1"
+  Then I should see "Loading..."
```
Comment (townsend3721) [inline features/guests_added.feature:87]: This is testing that Chartkick is loading up an interactive graph.  Unfortunately Capybara cannot interact with it beyond this.

File: features/guests_added.feature [hunk 2]
```diff
@@ -11,15 +11,46 @@ Feature: See if you are attending an event
...
+    When I follow "rsvp_1"
+    Then I should see "Gender"
+    And I fill out the form with the following attributes:
+      | firstname    | Test           |
+      | lastname     | Guest          |
+      | email        | test@guest.com |
+    Then I should see "You have successfully registered!"
+    And I should see "Test Guest"
+
+  Scenario: Delete a guest
```
Comment (harperiscoding) [inline features/guests_added.feature:26]: scenario does not match test so idk what this does

File: features/guests_added.feature [hunk 3]
```diff
@@ -11,15 +11,46 @@ Feature: See if you are attending an event
...
 
+    Given I am on the events page
+    When I follow "event_1"
+    Then I should see "Delete"
+    And I follow "Delete"
+    Then I should see "Guest was successfully removed."
+
+  Scenario: I register for an event with demographics:
+    When I follow "rsvp_1"
+    And I select "Undergrad" from "guest[occupation]"
```
Comment (harperiscoding) [inline features/guests_added.feature:44]: did not fix this.

File: features/guests_added.feature (hunk 1 above)
Comment (harperiscoding) [reply features/guests_added.feature:87]: works for me

File: app/views/events/show.html.erb [hunk 4]
```diff
@@ -27,6 +27,12 @@
   <strong>Capacity:</strong>
   <%= @event.capacity %>
 </p>
+<p>
+  <strong>Demographics:</strong>
```
Comment (harperiscoding) [inline app/views/events/show.html.erb:31]: still need to be hidden unless you're logged in.

File: db/migrate/20190401024406_create_guests.rb [hunk 5]
```diff
@@ -4,8 +4,10 @@ def change
       t.string :firstname
       t.string :lastname
       t.string :email
+      t.string :occupation
```
Comment (harperiscoding) [inline db/migrate/20190401024406_create_guests.rb:7]: again, migration is bad form.

**Review by harperiscoding — CHANGES_REQUESTED** [review]

File: features/guests_added.feature (hunk 2 above)
Comment (townsend3721) [reply features/guests_added.feature:26]: Read the entire scenario.  There is just an extra space in between so it's easier to read

File: features/guests_added.feature (hunk 3 above)
Comment (townsend3721) [reply features/guests_added.feature:44]: Fill out the form doesn't work with select boxes.  That's why I had to manually select each one.

Comment (harperiscoding) [reply features/guests_added.feature:44]: then create a step.

File: app/views/events/show.html.erb (hunk 4 above)
Comment (townsend3721) [reply app/views/events/show.html.erb:31]: See comment from last closed pull request

File: db/migrate/20190401024406_create_guests.rb (hunk 5 above)
Comment (townsend3721) [reply db/migrate/20190401024406_create_guests.rb:7]: working on it

File: features/guests_added.feature (hunk 2 above)
Comment (harperiscoding) [reply features/guests_added.feature:26]: you need to create a new scenario or add this to step def cause the first half does not test if a guest is deleted.

Comment (townsend3721) [reply features/guests_added.feature:26]: This is not testing deleting a guest, this is testing registering with no demographics.  Look below for the delete scenario.

Comment (harperiscoding) [reply features/guests_added.feature:26]: Hence my point: the scenario is for deleting a guest. review how to write/organize cucumber tests.

File: app/views/events/show.html.erb (hunk 4 above)
Comment (harperiscoding) [reply app/views/events/show.html.erb:31]: Kiera specifically said they need to be hidden unless you're logged in

Comment (townsend3721) [reply app/views/events/show.html.erb:31]: She didn't want people's personal information available to the public, so I think this is different.  I will contact her about it

Comment (harperiscoding) [reply app/views/events/show.html.erb:31]: this is not different, she specifically said she didn't want the demographic information to be public. Don't contact her, that's unnecessary considering we had a meeting yesterday where she specifically said she didn't want demographic information to be public.

**Pull request closed without merging**